envoy/upstream/upstream.h

```diff
@@ -743,6 +743,7 @@ class PrioritySet {
   COUNTER(upstream_rq_cancelled)                                                                   \
   COUNTER(upstream_rq_completed)                                                                   \
   COUNTER(upstream_rq_maintenance_mode)                                                            \
+  COUNTER(upstream_rq_drop_overload)                                                               \
```

Comment:
This stat will need to be added also to the docs.

Reply:
After checking this PR: https://github.com/envoyproxy/envoy/pull/3894 and this comment:
https://github.com/envoyproxy/envoy/blob/4d154dd245690cfcf24fa21ad395f86800724d71/envoy/upstream/upstream.h#L774, I moved the counter from the cluster trafficStats to loadReportStats to align with its purpose.  Please  comment if you have different opinion.